### ISSUE TITLE
React admin 3.0.0-beta.3 with IE11: ReferenceError: 'Proxy' is undefined

### ISSUE THREAD

**Issue by panfiva** [marmelab/react-admin#3881]
I followed user guide to create a sample application (https://github.com/marmelab/react-admin/blob/next/docs/Tutorial.md). To enable IE support, I added the following statements into src\index.js:

import 'react-app-polyfill/ie11'
import 'react-app-polyfill/stable';

When accessing an application, I see menu and navbar rendering properly, however instead of data table, I see an error message "Something went wrong" with the following details (see screen print; source code is also attached). Is there a way to enable IE support in v3?

==================
ReferenceError: 'Proxy' is undefined

    in List (at users.js:6)
    in UserList (created by WithPermissions)
    in WithPermissions (created by Context.Consumer)
    in Route (created by ResourceRoutes)
    in Switch (created by ResourceRoutes)
    in ResourceRoutes (created by Resource)
    in Resource (at App.js:10)
    in Route (created by RoutesWithLayout)
    in Switch (created by RoutesWithLayout)
    in RoutesWithLayout (created by Context.Consumer)
    in div (created by Layout)
    in main (created by Layout)
    in div (created by Layout)
    in div (created by Layout)
    in Layout (created by WithStyles(Layout))
    in WithStyles(Layout) (created by Context.Consumer)
    in withRouter(WithStyles(Layout)) (created by ConnectFunction)
    in ConnectFunction (created by LayoutWithTheme)
    in ThemeProvider (created by LayoutWithTheme)
    in LayoutWithTheme (created by Context.Consumer)
    in Route (created by CoreAdminRouter)
    in Switch (created by CoreAdminRouter)
    in div (created by CoreAdminRouter)
    in CoreAdminRouter (created by Context.Consumer)
    in Route (created by CoreAdmin)
    in Switch (created by CoreAdmin)
    in Router (created by ConnectedRouter)
    in ConnectedRouter (created by Context.Consumer)
    in ConnectedRouterWithContext (created by ConnectFunction)
    in ConnectFunction (created by CoreAdmin)
    in TranslationProvider (created by CoreAdmin)
    in Provider (created by CoreAdmin)
    in CoreAdmin (at App.js:9)
    in App (at src/index.js:14)

=================================

![image](https://user-images.githubusercontent.com/48455289/67576525-589b1880-f704-11e9-8cb3-9748817527c7.png)

Attached source code

[react-admin-app-3.zip](https://github.com/marmelab/react-admin/files/3772354/react-admin-app-3.zip)

**Comment by fzaninotto** [comment]
Confirmed. `Proxy` isn't supported by IE 11

![image](https://user-images.githubusercontent.com/99944/67666996-3a842100-f96d-11e9-82f6-6f9e2dbd71fe.png)

We must document that IE11 support requires adding a polyfill, like https://github.com/GoogleChrome/proxy-polyfill

**Comment by panfiva** [comment]
I tried making it work with proxy-polyfill but did not have much luck, can step-by-step instructions can be added? thanks

**Comment by fzaninotto** [comment]
Sorry, I don't have an IE11 nearby. Does https://github.com/GoogleChrome/proxy-polyfill/pull/58/files help?

**Comment by fzaninotto** [comment]
I didn't manage to properly configure the example to compile for IE11, so we'll leave it to the community to help debug this one. 

I think the Google Proxy polyfill is the solution, but I don't know the right syntax (their doc is outdated) between

```js
import 'proxy-polyfill/proxy.min.js';
```

and

```js
const ProxyFactory = require('proxy-polyfill/proxy.min.js');
const PProxy = ProxyFactory();
```

**Comment by m4theushw** [comment]
I managed to run the demo on IE11 by including the following polyfills in examples/demo/src/index.js:

```js
import 'react-app-polyfill/ie11';
import 'react-app-polyfill/stable';
import 'proxy-polyfill';
```

![ie11](https://user-images.githubusercontent.com/42154031/69170343-b428ae00-0ad8-11ea-9e4b-1e9885830e15.png)

PR #4007 adds these changes to the demo, feel free to change or close it.

**Comment by fzaninotto** [comment]
Fixed by #4007.
